Translations update from OSGeo Weblate

osgeoweblate opened the pull request:
Translations update from [OSGeo Weblate](https://weblate.osgeo.org) for [GTT Project/Redmine Custom Fields Groups plugin](https://weblate.osgeo.org/projects/gtt-project/redmine_custom_fields_groups/).



Current translation status:

![Weblate translation status](https://weblate.osgeo.org/widgets/gtt-project/-/redmine_custom_fields_groups/horizontal-auto.svg)

weblate committed:
Added translation using OSGeo Weblate (German)

dkastl committed:
Translated using OSGeo Weblate (German)

Currently translated at 100.0% (12 of 12 strings)

Translation: GTT Project/Redmine Custom Fields Groups plugin
Translate-URL: https://weblate.osgeo.org/projects/gtt-project/redmine_custom_fields_groups/de/

dkastl committed:
Translated using OSGeo Weblate (Japanese)

Currently translated at 100.0% (12 of 12 strings)

Translation: GTT Project/Redmine Custom Fields Groups plugin
Translate-URL: https://weblate.osgeo.org/projects/gtt-project/redmine_custom_fields_groups/ja/

sanak commented on config/locales/ja.yml:
```diff
@@ -11,4 +11,4 @@ ja:
   label_fieldset_state_all_expended: 全て展開
   label_fieldset_state_all_collapsed: 全て折りたたみ
-  error_can_not_delete_custom_fields_group: カスタムフィールドグループを削除できません。
+  error_can_not_delete_custom_fields_group: カスタムフィールドグループを削除できません
```
Well, current translation comes from the following Redmine core side, so I will keep last `。` as well on Weblate side.
https://github.com/redmine/redmine/blob/5.1-stable/config/locales/ja.yml#L213
```yml
  error_can_not_delete_custom_field: 'カスタムフィールドを削除できません。'
```
https://github.com/redmine/redmine/blob/5.1-stable/config/locales/en.yml#L206
```yml
error_can_not_delete_custom_field: Unable to delete custom field
```

sanak replied:
Fixed above and dismissed the warning on Weblate.

sanak committed:
Translated using OSGeo Weblate (Japanese)

Currently translated at 100.0% (12 of 12 strings)

Translation: GTT Project/Redmine Custom Fields Groups plugin
Translate-URL: https://weblate.osgeo.org/projects/gtt-project/redmine_custom_fields_groups/ja/

sanak commented:
@dkastl Thanks a lot!
I merge this.
